content/zh/docs/installing-on-kubernetes/introduction/overview.md

```diff
@@ -27,40 +27,40 @@ weight: 4110
 
 ## 部署 KubeSphere
 
-确保现有的 Kubernetes 群集满足所有要求之后，可以使用 kubectl 执行 KubeSphere 的默认最小安装。
+确保现有的 Kubernetes 群集满足所有要求之后，您可以使用 kubectl 以默认最小安装包来安装 KubeSphere。
 
-- 执行以下命令以开始安装：
+1. 执行以下命令以开始安装：
 
-    ```bash
-    kubectl apply -f https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/kubesphere-installer.yaml
+```bash
+kubectl apply -f https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/kubesphere-installer.yaml
 
-    kubectl apply -f https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/cluster-configuration.yaml
-    ```
+kubectl apply -f https://github.com/kubesphere/ks-installer/releases/download/v3.0.0/cluster-configuration.yaml
+```
```

Comment:
Do not change the format here. The code should be aligned with the character, not the number. See the original English source text as reference.